docs/sources/administration/set-up-for-high-availability.md

```diff
@@ -22,7 +22,15 @@ Grafana will now persist all long term data in the database. How to configure th
 
 ## Alerting
 
-Currently alerting supports a limited form of high availability. [Alert notifications]({{< relref "../alerting/old-alerting/notifications.md" >}}) are deduplicated when running multiple servers. This means all alerts are executed on every server but alert notifications are only sent once per alert. Grafana does not support load distribution between servers.
+**Grafana 8 alerts**
+
+Grafana 8 Alerts provides a new highly-available model under the hood. It preserves some of the previous semantics by executing all alerts on every server. However, notifications are sent only once per alert. There is no support for load distribution between servers at this time.
```

Comment:
> It preserves some of the previous semantics by executing all alerts on every server. However, notifications are sent only once per alert.

To me the phrasing of this sentence suggests that old alerting cannot deduplicate notifications as it was not one of the previous semantics. However, in the description of old alerting below deduplication of notifications is mentioned as a supported feature?

Reply:
You're right, I have replaced However with and in this case.